spec/machine/RISCV64/MachineOps.thy

```diff
@@ -84,10 +84,6 @@ consts' timerPrecision :: "64 word"
 consts' max_ticks_to_us :: "64 word"
 consts' max_us_to_ticks :: "64 word"
 
-end
-
-qualify RISCV64 (in Arch)
```

Comment:
Due to abbreviations inside a `qualify` block seeming not to be properly qualified and then causing problems when being requalified, I moved the following types, abbreviation and definitions into the preceding `Arch` context. This makes all of the qualifications work, but does annoyingly mean that uses of the definitions inside of the `qualify` block need to themselves be qualified. This is the best approach that we were able to find for this.